### PR TITLE
Add the Ultrasonic sensor back

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

**Review by ErikBoesen — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -36,4 +49,9 @@ public void shoot(double shooterSpeed) {
         // shooterController.setReference(shooterSpeed, ControlType.kVelocity);
         shooterMotor1.set(shooterSpeed);
     }
+
+    @Override
+    public void periodic() {
+        rangeFilter.calculate(ballSensor.getRangeInches());
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:75]: What's the point of calculating this every loop if you calculate it again when you call `isBallReady`

Comment (AndrewLester) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:75]: Calculating it in each loop makes sure that the `rangeFiler` has an up-to-date distance value, as it takes a few `calculate` calls for it to update to the true value. I would actually just not call `calculate` in `isBallReady` in favor of some `getFilteredValue` method, but I don't think that exists.

Comment (ArchdukeTim) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java:75]: Oh I see. Ok gotcha

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -2,27 +2,40 @@
...
         shooterMotor1.setInverted(true);
         shooterMotor2.follow(shooterMotor1);
     }
 
+    public boolean isBallReady() {
+        return rangeFilter.calculate(ballSensor.getRangeInches()) <= BALL_EXISTS_DISTANCE;
+    }
+
     public void activatePiston() {
         shooterSolenoid.set(true);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java:57]: You should probably reset the filter here

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -27,6 +27,8 @@
     public static final int SHOOTER_MOTOR_1 = 10;
     public static final int SHOOTER_MOTOR_2 = 12;
     public static final int SHOOTER_SOLENOID_PORT = 0;
+    public static final int SHOOTER_ULTRASONIC_TRIG = 3;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/Constants.java]: the var name here is `TRIG` but the wpilib api has the first port being for ping and the second for echo. Why not call this `SHOOTER_ULTRASONIC_PING`? What does `TRIG` stand for?

**Review by ArchdukeTim — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 3 above)
Comment (AndrewLester) [reply src/main/java/frc/robot/Constants.java]: I think trig and ping mean the same thing in this case, but the port on the actual device itself is labeled `trig`. I'm guessing it means trigger, but I'm not entirely sure.

**Pull request merged**